apps/src/lib/util/AnalyticsConstants.js

```diff
@@ -47,6 +47,7 @@ const EVENTS = {
   CAP_PARENT_EMAIL_MODAL_UPDATED: 'CAP Parent Email Modal Updated',
   CAP_PARENT_EMAIL_MODAL_RESEND: 'CAP Parent Email Modal Resend',
   CAP_PARENT_EMAIL_MODAL_CLOSED: 'CAP Parent Email Modal Closed',
+  CAP_AGE_GATED_MODAL_SHOWN: 'CAP Age Gated Modal Shown',
```

Comment:
I think there area few metrics missing: 
1. Banner Shown (this was missing from the JIRA originally).
2. Button in banner clicked.
3. Documentation link clicked.

I realize the button being clicked and the modal would be redundant, but lets do this for now just incase we change behavior in the future (such as auto showing the modal).

I have updated the [JIRA](https://codedotorg.atlassian.net/browse/P20-904) task to only include the work being done in this PR and added the metric.

Also could you also make sure the metric event name matches what is in the JIRA? I have updated them to follow the pattern in this code.

Reply:
I plan on adding the Banner metrics in the following [PR](https://github.com/code-dot-org/code-dot-org/pull/58481).